e2e-fixtures/hello-https-ingress/hello-https-ingress.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: minimal-ingress-https
+  annotations:
```

Comment:
The desire in the slack thread was for it to be a property of the service, not the ingress, modeling after how a google load balancer does it. Which I think does make sense, if you have several ingress configs to the same service, you only configure that the service wants TLS once, can't get out of sync.

Reply:
Turns out after some discussion that I completely misinterpreted the comment. I've pushed a change that removes the `Ingress` annotations entirely, since they were just copied from another fixture and are unnecessary for what's going on in this PR.